Cython/Compiler/PyrexTypes.py

```diff
@@ -1691,6 +1691,27 @@ def get_fused_types(self, result=None, seen=None):
             seen.add(self)
 
 
+class ConstFusedType(FusedType):
```

Comment:
Why is this additional type necessary, instead of relying on the `CConstOrVolatileType` wrapper for this?
Using just the wrapper would be nicer because it's already integrated into the type system and also handles more cases.

Reply:
Indeed.
Adding `subtypes = ['cv_base_type']` to `CConstOrVolatileType` makes it compatible with `FusedType` and boils down this patch to a single liner...